### ISSUE TITLE
edit link for GitLab

### ISSUE THREAD

**Issue by noraj** [retypeapp/retype#44]
```
WARNING: [retype.json] Only edit on GitHub is supported.
         Please use --verbose for more details
```

Adding edit support for GitLab too

**Comment by geoffreymcgill** [comment]
We are going to remove this `github.com` only limitation and will support `gitlab.com` repos as well as any other custom `edit` paths. The change is being made today and will be included in the upcoming Retype `v1.7.0` release, which might also be released later today or tomorrow.

Hope this helps.

**Comment by geoffreymcgill** [comment]
This enhancement to the `edit` logic is complete and will be included in `v1.7.0`, which should be released tomorrow.

Closing this issue but feel free to follow-up if any further defects are discovered after the `v1.7.0` release.
